incubator/patroni/templates/statefulset-patroni.yaml

```diff
@@ -141,6 +141,9 @@ spec:
       tolerations:
 {{ toYaml . | indent 8 }}
     {{- end }}
+    {{- with .Values.schedulerName }}
+      schedulerName: {{ . }}
+    {{- end }}
```

Comment:
Most charts seem to use `{{- if .Values.schedulerName }}` instead of `with`. https://github.com/helm/charts/blob/0f284f7d959e7b92501e138318983f866828ff4f/stable/mariadb/templates/master-statefulset.yaml#L38-L40

Can we do it similar here?

Reply:
`with` seems cleaner to me as it cuts down on repetition. is the idea that `if` is more explicit and thus easier for users to understand? or is it purely a matter of consistency?

Reply:
I'm fine with it if the behaviour is the same. Does `with` imply the `if`?

Reply:
oh whoops, i already pushed your suggested change. yep, `with` won't render if its value is falsey.

btw, while we discuss this, can you `/ok-to-test`? I'm not a Collaborator so this PR's CI tests are blocked until approved for testing

Reply:
Thanks for clarifying this!